.github/workflows/maven.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  install:
+    strategy:
+      matrix:
+        platform: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 50
```

Comment:
Travis use to do all the commits in the PR. I don't know what github workflow will do so I made this a bigger number for now.